vscode-trace-extension/src/extension.ts

```diff
@@ -51,6 +51,15 @@ export function activate(context: vscode.ExtensionContext): void {
         if (e.affectsConfiguration('trace-compass.traceserver.url') || e.affectsConfiguration('trace-compass.traceserver.apiPath')) {
             updateTspClient();
         }
+
+        if (e.affectsConfiguration('trace-compass.traceserver.url')) {
+            // Signal the change to the `Opened traces` and `Available views` webview
+            tracesProvider.updateTraceServerUrl();
+            myAnalysisProvider.updateTraceServerUrl();
+
+            // Signal the change to all trace panels
+            TraceViewerPanel.updateTraceServerUrl();
+        }
```

Comment:
It looks odd that the `updateTraceServerUrl` methods don't accept the new value. How do we guarantee that they all refer to the same value in the end?

Reply:
We have a function called `getTraceServerUrl()` in the `vscode-trace-extension\src\utils\tspClient.ts` that gets the latest url and perform some additional formatting. I use this function inside each `updateTraceServerUrl()` function to get the url.

Reply:
Is there a reason to leave the separate implementations to call that function, rather than calling it once here and passing the value in?

Reply:
There is no particular reasons. For clarity, I think your suggestion is easier to read/maintain. I will make the adjustment.